src/conn/conn_layered.c

```diff
@@ -1047,44 +1010,27 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
         WT_ERR(__wt_config_gets(session, cfg, "disaggregated.checkpoint_meta", &cval));
         if (cval.len > 0) {
             WT_WITH_CHECKPOINT_LOCK(
-              session, ret = __disagg_pick_up_checkpoint_meta(session, &cval, &checkpoint_id));
+              session, ret = __disagg_pick_up_checkpoint_meta(session, &cval));
             WT_ERR(ret);
-        } else {
-            WT_ERR(__wt_config_gets(session, cfg, "disaggregated.checkpoint_id", &cval));
-            if (cval.len > 0 && cval.val >= 0) {
-                checkpoint_id = (uint64_t)cval.val;
-                WT_WITH_CHECKPOINT_LOCK(
-                  session, ret = __disagg_pick_up_checkpoint(session, 0, checkpoint_id));
-                WT_ERR(ret);
-            } else
-                /*
-                 * FIXME-WT-14731: If we are starting with local files, get the checkpoint ID from
-                 * them? Alternatively, maybe we should just fail if the checkpoint ID is not
```

Comment:
Perhaps we can resolve WT-14731?  I put a comment in that ticket to take a look at that.